src/ui/pages/IncomingRequest/components/BallotTransactionRequest.scss

```diff
@@ -0,0 +1,88 @@
+.ballot-transaction {
+  & .page-header {
+    ion-toolbar {
+      --background: transparent;
+
+      ion-title.md {
+        width: calc(100% - 3.75rem);
+      }
+    }
+  }
```

Comment:
Since we are using the `PageHeader` component, if you notice something that needs adjustments please check if the same component needs adjustments also on other pages and if you notice something in common maybe it's better if we make those adjustments in the `PageHeader` component itself rather than keep adding ad-hoc changes to every other file that is using it.

Same with the title, the size should be already set by the `PageHeader` component. Please make sure it works fine in all situations without us making adjustments.